.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: trusty
```

Comment:
can we avoid specifying the dist?

Reply:
I added the dist command based on documentation by travis CI. Its seems that linux 12 has problems with chrome (headless) and this command ensure that linux 14 is used. If linux 14 is used by default I can remove it.

Reply:
pretty sure that trusty is the default now (we upgraded a few weeks ago)

Reply:
ok good to know, will remove it.